include/Song.h

```diff
@@ -375,6 +376,8 @@ class LMMS_EXPORT Song : public TrackContainer
 
 	const std::string& syncKey() const noexcept { return m_vstSyncController.sharedMemoryKey(); }
 
+	Metronome& metronome();
```

Comment:
I don't think we have a rule about this, but I prefer to put trivial getters and setters in the header. It's a little more concise, and allows them to be inlined across translation units.